backend/routes/index.ts

```diff
@@ -21,17 +21,28 @@ router.post('/signup', AuthController.signUp);
 router.post('/signin', AuthController.signIn);
 router.post('/forgot-password', AuthController.forgotPassword);
 
-
 router.get('/locations/:id', LocationsController.getLocationById);
+router.patch('/locations/:id', LocationsController.updateLocationById);
 router.get('/locations/', LocationsController.getLocationsByZoom);
```

Comment:
name of controller function and route should be changed to avoid route confusion
e.g.
updateLocationRating
and 
/locations/rating-update/:id

Reply:
We don't need to create a new route for this. We can update the entire location at once.